packages/ui/pages/api/chain/[chainId]/comptroller/[comptrollerAddress].ts

```diff
@@ -0,0 +1,38 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import * as yup from 'yup';
+
+import { ALL_POOLS_INFO, SUPPORTED_NETWORKS_REGEX, VALID_ADDRESS_REGEX } from '@ui/constants/index';
+
+export type ResponseType = {
```

Comment:
Lets call it `GetComptrollerResponse`

As it's all in the same nextjs project, you should be able to use this type in the UI code as well.

Reply:
Fixed